README.md

```diff
@@ -1 +1,102 @@
-# ovirt-ansible-engine-setup
+oVirt Engine Setup
+==================
+
+Install required packages for oVirt Engine deployment, generates answerfile
+and runs engine-setup with it.
+Optionally the role can update oVirt engine packages.
+
+Target Systems
+--------------
+
+* engine
+
+Requirements
+------------
+
+ * Environment with configured repositories.
+ * Ansible version 2.4
+
+Role Variables
+--------------
+
+By default engine-setup uses answer file specific for version of oVirt,
+based on ``ovirt_engine_setup_version`` parameter. You can specify own answer file
+as ``ovirt_engine_setup_answer_file_path``.
+
+* Common options:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_version            | 4.2                   | Allowed versions: [3.6, 4.0, 4.1, 4.2] |
+| ovirt_engine_setup_product_type       | ovirt-engine          | Type of product 'ovirt-engine' or 'rhvm'  |
+| ovirt_engine_setup_package_list       | []                    |  List of extra packages to be installed on engine apart from ovirt-engine package. |
+| ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
+| ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
+| ovirt_engine_setup_firewall_manager   | firewalld             | Specify the type of firewall manager to configure on Engine host, following values are availableL: `firewalld`,`iptables` or empty value to skip firewall configuration. |
+| ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
+| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. |
+| ovirt_engine_setup_accept_defaults    | False                 | If `True` use option `--accept-defaults` when executing `engine-setup`. |
+| ovirt_engine_setup_require_rollback   | UNDEF                 | If `True` setup will require to be able to rollback new packages in case of a failure. If not passed the default answer from `engine-setup` will be chosen. Valid for updating/upgrading.
```

Comment:
I do not understand this. Isn't the default answer chosen only if using "accept_defaults"? If not, engine-setup should prompt. Do we then let the user answer?

Reply:
Well, I have in the ansewr fille,  OSETUP_RPMDISTRO/requireRollback=none:None in case user doesn't give answer, so default value will be chosen, without the use of accept-defaults.
What we have decided is that all answers should exists in the answer file.
Accept defaults makes sense for example when custom answer file will be used.